addons/mail/wizard/mail_template_preview.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class MailTemplatePreview(models.TransientModel):
+    _inherit = "mail.template"
+    _name = "mail.template.preview"
+    _description = "Email Template Preview"
+
+    @api.model
+    def _selection_target_model(self):
+        models = self.env['ir.model'].search([])
+        return [(model.model, model.name) for model in models]
```

Comment:
`return [(model.model, model.name) for model in self.env['ir.model'].search([])]`